components/Companies/CompanyModal/elements.tsx

```
@@ -0,0 +1,86 @@
+}: CompanyModalFieldsProps) => {
+  return (
+    <ModalBody>
+      <Input
+        className="pt-2"
+        errorMessage={errors?.name?.message || ""}
+        isInvalid={!!errors.name}
+        label="Name"
+        labelPlacement="outside"
+        placeholder="Company name"
+        size="lg"
+        variant={!!errors.name ? "bordered" : "flat"}
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)